docs/_docs/swarm/run.md

```diff
@@ -0,0 +1,140 @@
+---
+nav_order: 2
+redirect_from: /docs/swarm/run.html
+---
+
+# Run
+
+***
+
+The run command of the HiveMQ Swarm command line offers a set of commands to start and stop runs.
+
+```
+mqtt swarm run
+```
+
+***
+
+## Commands
+
+| Command | Explanation             |
+|---------|-------------------------|
+| start   | See [start](#start-run) |
+| stop    | See [stop](#stop-run)   |
+
+***
+
+## Options
+
+| Option | Long Version | Explanation                                | Default                 |
+|--------|--------------|--------------------------------------------|-------------------------|
+| `-url` |              | The URL of the HiveMQ Swarm Rest endpoint. | `http://localhost:8888` |
+| `-r`   | `--run-id`   | The id of the run to stop.                 | `The current run`       |
```

Comment:
```suggestion
| `-r`   | `--run-id`   | The id of the run.                 | `The current run`       |
```
if I understand this correctly, you can set the run-id on start, and use it to stop a specific run as well, right?

Reply:
Yes!

Reply:
You are correct. Will remove the row